docs/help-topics/appstudio-incontext-help/metadata.yml

```diff
@@ -0,0 +1,3 @@
+kind: HelpTopic
+name: appstudio-incontext
+# tags:
```

Comment:
Tags are not mandatory-you can remove the key. But add an empty line to the end of the file, please.

Reply:
Will do, thank you. Didn't know .yml requires that too, we add empty lines in .adoc.

Reply:
It's just a convention. Its not strictly "required"